tests/unit/test_common.cpp

```diff
@@ -243,3 +247,52 @@ TEST_F(CommonTest, Max_Updatable_Heap) {
     ASSERT_EQ(heap.size(), 0);
     ASSERT_TRUE(heap.empty());
 }
+
+TEST_F(CommonTest, VecSim_Normalize_FP32_Vector) {
+    const size_t dim = 1000;
+    float v[dim];
+
+    // generate random values - always generates the same values
```

Comment:
please set the random seed to be constant to do so
check other tests to see how